redux/README.md

```diff
@@ -0,0 +1,34 @@
+# Managing application state - redux
+
+## Agenda
+
+1. Problem introduction
+  - Components are [too long](https://github.com/msd-code-academy/06-cinema-portal-redux/blob/master/src/PurchasePage.js)
+  - We need pass props (data or event handlers) up and down
+  - We don't have enough abstractions for app to grow
+
+1. Time-travel debugging in Redux devtools
```

Comment:
does not render as ordered list on https://github.com/msd-code-academy/lessons/blob/10153ba9a230646aa6ff8f9fe5b8c693777e4230/redux/README.md => can use use 2. ?